.gitignore

```diff
@@ -104,3 +104,5 @@ dist
 .tern-port
 .vscode/arduino.json
 .vscode/c_cpp_properties.json
+
+.idea/
```

Comment:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Please do that instead.